rdkit/Chem/UnitTestFeatFinderCLI.py

```diff
@@ -0,0 +1,69 @@
+from contextlib import contextmanager
+import os
+import sys
+import unittest
+
+from rdkit import RDConfig
+from rdkit.Chem import FeatFinderCLI
+from rdkit.six.moves import cStringIO as StringIO
+
+
+class TestCase(unittest.TestCase):
+
+  def test_FeatFinderCLI(self):
+    smilesFile = os.path.join(RDConfig.RDDataDir, 'NCI', 'first_5K.smi')
+    featureFile = os.path.join(RDConfig.RDCodeDir, 'Chem', 'Pharm2D', 'test_data',
+                               'BaseFeatures.fdef')
+    parser = FeatFinderCLI.initParser()
+    cmd = '-n 10 {0} {1}'.format(featureFile, smilesFile)
+    with outputRedirect() as (out, err):
+      args = parser.parse_args(cmd.split())
+      FeatFinderCLI.processArgs(args, parser)
+    self.assertIn('Mol-1', out.getvalue())
```

Comment:
Not sure if this is reasonable or not, but the repeated calls to out.getvalue() seem strange to me. Wouldn't it be better to read that once into a variable and then test against that?